prow/plugins/config.go

```diff
@@ -427,10 +428,21 @@ type ConfigMapSpec struct {
 	// Clusters is a map from cluster to namespaces
 	// which specifies the targets the configMap needs to be deployed, i.e., each namespace in map[cluster]
 	Clusters map[string][]string `json:"clusters"`
+	// ClusterGroup is a list of named cluster_groups to target. Mutually exclusive with clusters.
```

Comment:
It seems like you could achieve this same effect with a yaml anchor and keep this complexity out of prow's codebase.

Is there a reason to prefer moving this complexity inside prow?

Reply:
From my understanding, yaml anchors do not work for maps: https://github.com/go-yaml/yaml/issues/184

Generally, I am happy with anything that works